universal-application-tool-0.0.1/app/controllers/admin/QuestionController.java

```diff
@@ -102,6 +107,22 @@ public QuestionController(
             httpExecutionContext.current());
   }
 
+  @Secure(authorizers = Authorizers.Labels.UAT_ADMIN)
+  public CompletionStage<Result> view(Request request, Long id) {
```

Comment:
Question (blocking): what's different between the form rendered by this method and the one rendered by the `edit` method in this class?

Reply:
Ah got, please use `show` for that: https://github.com/seattle-uat/civiform/wiki/Development-standards#html-routing-convention

Reply:
Done.